### PR TITLE
Add horn to cli

### PR DESCRIPTION
**Pull request by Severon96**
## Proposed change
For convenience it would be nice to have the possibility of triggering the horn via cli. Same as it's possible to flash the lights via cli or app it should be possible to horn via cli or app.

I added the function to the cli.py file the same way the light flashing is implemented.

## Type of change
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to this library)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Checklist
- [ ] The code change is tested and works locally.
- [ ] Tests have been added to verify that the new code works.
Regarding tests: I haven't seen api-tests so i did not add unit tests for this functionality. If you point me to some example tests I can add them, too.

### CODE REVIEW

File: bimmer_connected/cli.py [hunk 1]
```diff
@@ -46,6 +46,11 @@ def main_parser() -> argparse.ArgumentParser:
     flash_parser.add_argument("vin", help=TEXT_VIN)
     flash_parser.set_defaults(func=light_flash)
 
+    flash_parser = subparsers.add_parser("horn", description="Flash the vehicle lights.")
+    _add_default_arguments(flash_parser)
+    flash_parser.add_argument("vin", help=TEXT_VIN)
+    flash_parser.set_defaults(func=horn)
```
Comment (rikroe) [inline bimmer_connected/cli.py:52]: Please rename to `horn_parser`. Should work anyway, but is much clearer.

**Review by rikroe — CHANGES_REQUESTED** [review]

File: bimmer_connected/cli.py (hunk 1 above)
Comment (Severon96) [reply bimmer_connected/cli.py]: @rikroe I adjusted the changes as wished. sorry, I oversaw the variable name.

File: bimmer_connected/cli.py [hunk 2]
```diff
@@ -46,6 +46,11 @@ def main_parser() -> argparse.ArgumentParser:
     flash_parser.add_argument("vin", help=TEXT_VIN)
     flash_parser.set_defaults(func=light_flash)
 
+    horn_parser = subparsers.add_parser("horn", description="Flash the vehicle lights.")
```
Comment (rikroe) [inline bimmer_connected/cli.py]: Please also update the description to `Sound the horn.`. Sorry, missed that one earlier as well.

Comment (gerard33) [inline bimmer_connected/cli.py:49]: The description still needs to be changed.

**Review by gerard33 — DISMISSED** [review]

**Comment by rikroe** [comment]
The mypy and sphinx tests are failing due to #495 getting merged and can be ignored.

File: bimmer_connected/cli.py (hunk 1 above)
Comment (Severon96) [reply bimmer_connected/cli.py]: Changed

File: bimmer_connected/cli.py (hunk 2 above)
Comment (Severon96) [reply bimmer_connected/cli.py]: Changed

**Review by rikroe — APPROVED** [review]

**Comment by rikroe** [comment]
LGTM, thanks!

**Pull request merged**